internal/api/auth/config.go

```diff
@@ -26,8 +36,14 @@ type RedisConfig struct {
 
 // Validate implements config.ConfigurationValidator.
 func (c Config) Validate() error {
-	if c.Enabled && c.OIDC == nil {
+	if !c.Enabled {
+		return nil
+	}
+	if c.OIDC == nil || c.OIDC.ProviderURL == "" || c.OIDC.ClientID == "" {
 		return errors.New("Missing OIDC configuration with API auth. enabled.")
 	}
+	if c.OIDC.ABAC.Enabled && c.OIDC.ABAC.ClaimPath == "" {
+		return errors.New("Missing OIDC ABAC claim path with ABAC enabled.")
```

Comment:
```suggestion
		return errors.New("missing OIDC ABAC claim path with ABAC enabled.")
```

https://google.github.io/styleguide/go/decisions.html#error-strings